client/src/components/generic/PageTitle/PageTitle.tsx

```diff
@@ -0,0 +1,14 @@
+export interface IPageTitleProps {
+  title: string
+}
+type props = IPageTitleProps
+
+export const PageTitle = ({ title }: props) => {
+  return (
+    <div className="bg-light-blue-100 flex h-[70px] w-full items-center justify-center">
+      <div>
```

Comment:
Is the div still needed with that change?

Reply:
I have a habit of wrapping everything into a div. Removed now.